setup.py

```diff
@@ -576,15 +571,6 @@ def check_file(f):
     include_dirs.insert(0, CUDNN_INCLUDE_DIR)
```

Comment:
Shouldn't cuDNN be already linked to ATen?

Reply:
Yes, but PyTorch also needs to link against CuDNN, because some codepaths, e.g., LSTM, go through this codepath.